appservice/src/SiteWrapper.ts

```diff
@@ -17,6 +17,15 @@ import { ArgumentError } from './errors';
 import * as FileUtilities from './FileUtilities';
 import { localize } from './localize';
 
+const noSupport: string = localize('noSupport', 'This deployment source is not currently supported and will be deployed by zip.');
```

Comment:
IMO the user should only see this message on deploy, instead of when they're changing the type

Reply:
Hmm, but I feel like as a user, I'd be annoyed to switch to GitHub, expecting to deploy, and finding out that it is not supported when I try to (possibly at a much later time).

Reply:
Maybe we don't show GitHub here at all. It seems like the user will have to enter additional information anyways:
<img width="634" alt="screen shot 2017-11-15 at 5 27 59 pm" src="https://user-images.githubusercontent.com/11282622/32869170-6fd1e852-ca2a-11e7-8336-19ddf7be1774.png">

Eventually we should prompt the user for the same info the portal asks for, but that seems like a separate feature

Reply:
(Either way the message needs to be 'not supported **in VS Code**' - your current message makes it feel like Azure App Service as a whole doesn't support Github)

Reply:
I'm fine with just removing GitHub and BitbucketGit until they are supported.